### ISSUE TITLE
possible issue at: tensorflow_v2/notebooks/3_NeuralNetworks/autoencoder.ipynb

### ISSUE THREAD

**Issue by nucccc** [aymericdamien/TensorFlow-Examples#406]
Referring to the notebook:

https://github.com/aymericdamien/TensorFlow-Examples/blob/master/tensorflow_v2/notebooks/3_NeuralNetworks/autoencoder.ipynb

In the following code portion (8th code block of the notebook):

`# Optimization process. 
def run_optimization(x):
    # Wrap computation inside a GradientTape for automatic differentiation.
    with tf.GradientTape() as g:
        reconstructed_image = decoder(encoder(x))
        loss = mean_square(reconstructed_image, x)

    # Variables to update, i.e. trainable variables.
    trainable_variables = weights.values() + biases.values()
    
    # Compute gradients.
    gradients = g.gradient(loss, trainable_variables)
    
    # Update W and b following gradients.
    optimizer.apply_gradients(zip(gradients, trainable_variables))
    
    return loss`

the line:

`trainable_variables = weights.values() + biases.values()`

results to me in an error due to the impossibility of summing two python dict_values. I personally solved the issue by converting both dict_values to lists:

`trainable_variables = list(weights.values()) + list(biases.values())`

I hope my issue is of utility for this great repo! Thank you for the work

**Comment by xy-always** [comment]
已经收到您的邮件。

**Comment by xmingpyccnu** [comment]
您好！很开心收到您的邮件。我将在查看邮件后，尽快给您回复。

**Comment by gpendrys** [comment]
Your comment helped me. Thank you!

**Comment by xy-always** [comment]
已经收到您的邮件。

**Comment by Alexzhang369** [comment]
这是来自QQ邮箱的假期自动回复邮件。您好，您的邮件已经收到。谢谢您，祝顺利。

**Comment by jiandandema** [comment]
谢谢您给马坤发送邮件，我会及时查看的，谢谢，辛苦了
